index/scorch/scorch.go

```diff
@@ -658,6 +658,16 @@ func (s *Scorch) StatsMap() map[string]interface{} {
 	m["num_persister_nap_merger_break"] = m["TotPersisterMergerNapBreak"]
 	m["total_compaction_written_bytes"] = m["TotFileMergeWrittenBytes"]
 
+	// the bool stat `index_active` indicates whether the background routines
```

Comment:
@Thejas-bhat after a little more thought, would you change this to something a little less ambiguous - like `index_merger_active` perhaps?

Reply:
like my thinking behind this stat was to indicate whether the entire index activity is idle because it has reached a steady state. that would be mean that both persister and merger need to be in sync with the root (epoch wise) and be idle, so naming the stat `index_merger_active` would be misleading in my opinion since if the persister's epoch went ahead the stat value would be true even if merger is in sync with root (but even that would eventually end up doing some work some time later).

Reply:
For closure here, believe we've agreed upon `index_bgthreads_active` or of the like.